includes/sanitizers/class-amp-o2-player-sanitizer.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * Class AMP_O2_Player_Sanitizer
+ *
+ * @package AMP
+ */
+
+/**
+ * Class AMP_O2_Player_Sanitizer
+ *
+ * Converts <div class="vdb_player><script></script></div> embed to <amp-o2-player>
+ *
+ * @see https://www.ampproject.org/docs/reference/components/amp-o2-player
+ */
+class AMP_O2_Player_Sanitizer extends AMP_Base_Sanitizer {
+	const URL_PATTERN = '#.*delivery.vidible.tv\/jsonp\/pid=(.*)\/vid=(.*)\/(.*).js.*#i';
```

Comment:
Might want to use named capture groups to make it easier to reference the `$matches`.